Library/Formula/pinentry-mac.rb

```diff
@@ -13,9 +12,31 @@ class PinentryMac < Formula
     sha1 "4d80f3954ddb16618074d9d978f3a83aedddb8db" => :mountain_lion
   end
 
+  resource "core" do
+    url "https://raw.githubusercontent.com/GPGTools/GPGTools_Core/master/newBuildSystem/prepare-core.sh"
```

Comment:
Since this is checked against a hash, perhaps this should be: *https://raw.githubusercontent.com/GPGTools/GPGTools_Core/0cfb8fd7ee0104a2c542a8d5593f1e252ce44bf9/newBuildSystem/prepare-core.sh*
So that the formula still works after upstream changes.

As for the *core_clone* resource, perhaps that should just use git-master instead of a revision, as their script does.